stores/healthcareProfessionalsStore.ts

```diff
@@ -20,60 +16,87 @@ export const useHealthcareProfessionalsStore = defineStore(
         const healthcareProfessionalsData: Ref<HealthcareProfessional[]>
         = ref([])
         const selectedHealthcareProfessionalId: Ref<string> = ref('')
-        const selectedHealthcareProfessionalData: Ref<HealthcareProfessional | undefined> = ref()
-        const healthcareProfessionalSectionFields = reactive({
-            healthcareProfessionalNameArray: [] as Array<LocalizedNameInput>,
-            localizedFirstName: '' as string,
-            localizedLastName: '' as string,
-            localizedMiddleName: '' as string,
-            nameLocale: Locale.EnUs as Locale,
-            healthcareProfessionalAcceptedInsurances: [] as Array<Insurance>,
-            healthcareProfessionalDegrees: [] as Array<Degree>,
-            healthcareProfessionalSpecialties: [] as Array<Specialty>,
-            healthcareProfessionalLocales: [] as Array<Locale>
+        const healthcareProfessionalSectionFields = reactive<HealthcareProfessional>({
```

Comment:
naming nit: I still don't know what "section" this is a part of, but if it makes sense to ya'll, I'll allow it @ShizuokaTerrier @NabbeunNabi